src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -627,6 +625,23 @@ public void populateFieldsToGenerateInQuestion(FeedbackQuestionAttributes feedba
                 Assumption.fail("Course disappeared");
             }
             break;
+        case OWN_TEAM_MEMBERS_INCLUDING_SELF:
+        case OWN_TEAM_MEMBERS:
+            if (teamOfEntityDoingQuestion != null) {
+                List<StudentAttributes> teamMembers = studentsLogic.getStudentsForTeam(teamOfEntityDoingQuestion,
+                        feedbackQuestionAttributes.getCourseId());
+
+                if (generateOptionsFor == FeedbackParticipantType.OWN_TEAM_MEMBERS) {
+                    teamMembers.removeIf(teamMember -> teamMember.getEmail().equals(emailOfEntityDoingQuestion));
+                }
+
+                teamMembers.forEach(teamMember -> {
```

Comment:
Inline the lambda?

Reply:
@daongochieu2810 do you intend to adopt this suggestion? Would be good to address comments to give reviewers some resolution

Reply:
sorry I forgot. Yes I will make the changes